src/main/scala/mesosphere/marathon/core/task/tracker/TaskTracker.scala

```diff
@@ -39,6 +42,7 @@ object TaskTracker {
   /**
     * Contains all tasks grouped by app ID.
     */
+  // FIXME: Rename to TaskStatesByApp
```

Comment:
No, because that would cause havoc for reviews.

Let's stabilize the concepts in this PR and then bundle the renames in a separate PR. That was a pattern for review happiness in the past.